lesson-15/main.js

```diff
@@ -0,0 +1,183 @@
+const btnCounterRoomsMinus = document.getElementById('btnCounterRoomsMinus');
+const btnCounterRoomsPlus = document.getElementById('btnCounterRoomsPlus');
+const childAgeQuastion = document.getElementById('childAgeQuastion');
+const guestInput = document.getElementById('guestInput');
+btnCounterChildrenPlus.addEventListener('click', addChildren);
+btnCounterChildrenMinus.addEventListener('click', removeChildren);
+
+
+
+
+
+// слайдер для Home guests loves
+
+function loadSlider() {
+  $(".slick__wrapper").slick({
+    slidesToShow: 4,
+    slidesToScroll: 4,
+    infinite: true,
+    arrows: true,
+  });
+}
+
+async function getResponse() {
+const response = await fetch('https://fe-student-api.herokuapp.com/api/hotels/popular');
+const data = await response.json();
+ 
+ console.log(data)
+
+ const homesInfoPanel = document.querySelector(".favorite-wrapper");
+
+ data.forEach((item) => { const hotelsItem = document.createElement("div");
+ hotelsItem.classList.add("guests-Homes");
+ hotelsItem.innerHTML = `
+   <img class="img_hotel" src="${item.imageUrl}" alt="${item.name}">
+   <div class="text_hotel">${item.name}</div>
+   <div class="location">${item.city}, ${item.country}</div>`;
+ homesInfoPanel.append(hotelsItem);
+});
+$(document).ready(function () {
+  $(".slick__wrapper").slick({
+    slidesToShow: 4,
+    slidesToScroll: 4,
+    infinite: true,
+    arrows: true,
+  });
+});
+ }
+getResponse()
+
+
+
+
+const filter = document.getElementById("filterForm");
+
+guestInput.onclick = function () {
+  filter.classList.toggle("display");
+};
+
+
+const ageSelector = document.getElementById("childAgeQuastion");
+btnCounterChildrenPlus.addEventListener('click', addChildren);
+
+function addChildren() {
+  ageSelector.style.display = "flex";
+  const selectOptions = document.createElement("select");
+  selectOptions.classList.add("year-selector");
+  selectOptions.setAttribute("name", "year-selector");
+  selectOptions.id = "shildAge"
+  selectOptions.innerHTML = `<select >
+                  <option>1 years old</option>
+                  <option>2 years old</option>
+                  <option>3 years old</option>
+                  <option>4 years old</option>
+                  <option>5 years old</option>
+                  <option>6 years old</option>
+                  <option>7 years old</option>
+                  <option>8 years old</option>
+                  <option>9 years old</option>
+                  <option>10 years old</option>
+                  <option>11 years old</option>
+                  <option>12 years old</option>
+                  <option>13 years old</option>
+                  <option>14 years old</option>
+                  <option>15 years old</option>
+                  <option>16 years old</option>
+                  <option>17 years old</option>
+                </select>`;
+ageSelector.append(selectOptions);
+}
+const shildAge = document.getElementById('shildAge');
```

Comment:
I really recommend you install eslint
![image](https://user-images.githubusercontent.com/20189313/141701080-ded3e513-147e-4738-909e-768e90922be0.png)